src/actions/WindowActions.js

```diff
@@ -572,30 +572,24 @@ export function initWindow(windowType, docId, tabId, rowId = null, isAdvanced) {
         });
       } else if (rowId) {
         //Existing row document
-        return getData(
-          'window',
-          windowType,
-          docId,
-          tabId,
-          rowId,
-          null,
-          null,
-          isAdvanced
-        ).catch((e) => {
+        return getData({
+          entity: 'window',
```

Comment:
'window' should have been a constant defined somewhere within the application. Same for the https://github.com/metasfresh/metasfresh-webui-frontend/pull/2661/files/5698a41c28bc5af9b97e421dd63ef677995baf15..f8fa18e5ecfe13bf9247e7f4928b41ce7aaa614e#diff-3c6645363a75fd175c34c81561d92ffcR115

Reply:
Theoretically yes, it shall be a constant.
Practically the getData swiss knife method is poor design.
IMHO we shall have a dedicated method getting 'window' data, 'view' data, 'board' data etc.